src/databricks/labs/remorph/reconcile/recon_config.py

```diff
@@ -68,6 +73,34 @@ def get_type(self):
         return None
 
 
+@dataclass
+class TableThresholdModel(Enum):
+    MISMATCH = "mismatch"
+    MISSING_IN_SOURCE = "missing_in_source"
```

Comment:
Let us not support MISSING_IN_SOURCE and TARGE
```suggestion

```

Reply:
Done